api/Sources/Api/PairQL/SuperAdmin/QueryAdmins.swift

```diff
@@ -152,87 +152,89 @@ func osVersionName(_ osVersion: Semver?) -> String? {
 
 struct AdminQuery: CustomQueryable {
   static func query(bindings: [Postgres.Data]) -> SQL.Statement {
-    .init("""
```

Comment:
there's some noise in these diffs that i don't understand... can you check that you're on `swiftformat --version` of exactly `0.55.5`? if not, can you get on that version and reformat?